package/src/components/Channel/Channel.tsx

```diff
@@ -640,7 +640,8 @@ const ChannelWithContext = <
   }, [threadPropsExists]);
 
   const handleAppBackground = useCallback(() => {
-    if (channel) {
+    const channelDefaultCapabilties = channel?.data?.own_capabilities as Array<string>;
```

Comment:
```
  const channelDefaultCapabilties = channel?.data?.own_capabilities as Array<string>;
  ```
    
refrain from using casting please, in almost every case. This is very much bug-prone for example. 
    
  Better version below: 
    
  ```
  const channelDefaultCapabilties = channel?.data?.own_capabilities
  if (channel && channelDefaultCapabilties?.includes('send-typing-events')) { 
 
 }
  ```

Reply:
There was a reason why it was done. The type of 'own_capabilities' is unknown, unfortunately.

Reply:
I myself is not in favour of such approaches

Reply:
Fixed it myself